package.json

```diff
@@ -50,5 +50,8 @@
   },
   "directories": {
     "test": "test"
+  },
+  "publishConfig": {
```

Comment:
TravisCI recommended putting this in due to NPM thinking you are trying to publish a private build